src/main/metrics/push-gateway.ts

```diff
@@ -0,0 +1,41 @@
+import { Request } from '@automationcloud/request';
+import { inject, injectable, multiInject } from 'inversify';
+
+import { Config, config } from '../config';
+import { Logger } from '../logger';
+import { MetricsRegistry } from './registry';
+
+const DEFAULT_PUSH_GATE_URL = 'http://push-gateway.monitoring.svc.cluster.local:9091/metrics';
+@injectable()
+export class MetricsPushgateway {
+    constructor(
+        @multiInject(MetricsRegistry)
+        protected registries: MetricsRegistry[],
+        @inject(Config)
+        public config: Config,
+        @inject(Logger)
+        public logger: Logger,
+    ) {
+    }
+
+    @config({ default: DEFAULT_PUSH_GATE_URL }) PUSH_GATEWAY_URL!: string;
+
+    async push(job: string) {
```

Comment:
I'm actually not quite sure about pushing all of them. Maybe we should allow pushing individual metrics via `push(metric: Metric)` and then add `pushAll()` as well?

Also for `job: string` — let's create an utility method like `util.getAppDetails()` that would read `${process.cwd()}/package.json` and return app name and version. This way it can be used to avoid specifying this `job` here — but also will be useful elsewhere (e.g. to create a default `StatusRouter`)